jorvik/celery.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import
+
+import os
+import traceback
+from celery import Celery, group
+from celery.utils.log import get_task_logger
+from django.db import transaction
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'jorvik.settings')
+
+app = Celery('jorvik')
+app.config_from_object('django.conf:settings', namespace='CELERY')
+
+# Load task modules from all registered Django app configs.
+app.autodiscover_tasks()
+
+logger = get_task_logger(__name__)
+
+
+# noinspection PyUnusedLocal
+@app.on_after_configure.connect
+def setup_periodic_tasks(sender, **kwargs):
+    sender.add_periodic_task(10.0, process_queue.s(500), name='email queue processor')
+
+
+@app.task(ignore_result=True)
+def send(pk):
+    # noinspection PyBroadException
+    try:
+        from posta.models import Messaggio
+        return Messaggio.invia(pk)
+    except Exception as e:
+        logger.error(traceback.format_exc())
+        return ['FAIL: worker {}'.format(e)]
+
+
+def get_email_batch(limit=None):
```

Comment:
Queste funzioni sono strettamente relative alla posta. Possiamo tenerle all'interno del modulo `posta`? Per esempio, possiamo creare la configurazione generale di Celery in `jorvik/celery.py`, ma tenere la configurazione per la coda della posta e i task relativi in `posta/celery.py`.